src/main/java/com/htisolutions/poolref/controllers/CreateSubLeagueController.java

```diff
@@ -0,0 +1,34 @@
+package com.htisolutions.poolref.controllers;
+
+import com.htisolutions.poolref.entities.User;
+import com.htisolutions.poolref.services.UserService;
+import com.htisolutions.poolref.viewModels.CreateSubLeagueViewModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.*;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.security.Principal;
+
+@Controller
+@RequestMapping("/create-sub-league")
+public class CreateSubLeagueController {
```

Comment:
We don't need a separate controller for creating a sub league, instead there should be a "Create" action on the Sub league controller